src/plugins/misc/commands/jumbo.command.ts

```diff
@@ -0,0 +1,36 @@
+import {ChatInputCommandInteraction, inlineCode, PermissionFlagsBits} from "discord.js";
+import {SlashCommandBuilder, SlashCommandScope} from "../../../builders/SlashCommandBuilder";
+import {useChatCommand} from "../../../hooks/useChatCommand";
+import axios from 'axios';
+
+const builder = new SlashCommandBuilder()
+    .setName("jumbo")
+    .setDescription("Make G'day blow up an image.")
+    .addStringOption(option => option.setName("emoji").setDescription("Emoji to jumbo size.").setRequired(true))
+    .setScope(SlashCommandScope.MAIN_GUILD);
+
+useChatCommand(builder, async (interaction: ChatInputCommandInteraction) => {
+    let emoji = interaction.options.getString('emoji', true)?.trim();
```

Comment:
This was a bit puzzling but I think I got it. I set a `let` variable named `emojiURL` equal null, so I can make the change inside the if statement and still be able to keep `emoji` a const.

Reply:
The main issue is we are/were reusing a variable for 2 purposes, first it holds the value of user input, then it holds the URL of the final image. This can lead to relying on the variable for user input after it has already pivoted to the URL which is unexpected, and this was seen in the last iteration

Reply:
I see, it makes sense now that it's pointed out. Writing that one down lol